pkg/cmd/cli/backup/create.go

```diff
@@ -138,12 +143,77 @@ func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
 		return err
 	}
 
+	var sharedInformerFactory informers.SharedInformerFactory
+	var updates chan *api.Backup
+	if o.Wait {
+		stop := make(chan struct{})
+		defer close(stop)
+
+		updates = make(chan *api.Backup)
+
+		sharedInformerFactory = informers.NewFilteredSharedInformerFactory(arkClient, 0, f.Namespace(), nil)
+		sharedInformerFactory.Ark().V1().Backups().Informer().AddEventHandler(
+			cache.FilteringResourceEventHandler{
+				FilterFunc: func(obj interface{}) bool {
+					backup, ok := obj.(*api.Backup)
+					if !ok {
+						return false
+					}
+					return backup.Name == o.Name
+				},
+				Handler: cache.ResourceEventHandlerFuncs{
+					UpdateFunc: func(_, obj interface{}) {
+						backup, ok := obj.(*api.Backup)
+						if !ok {
+							return
+						}
+						updates <- backup
+					},
+					DeleteFunc: func(obj interface{}) {
+						backup, ok := obj.(*api.Backup)
+						if !ok {
+							return
+						}
+						updates <- backup
+					},
+				},
+			},
+		)
+		go sharedInformerFactory.Start(stop)
+	}
+
 	_, err = arkClient.ArkV1().Backups(backup.Namespace).Create(backup)
 	if err != nil {
 		return err
 	}
 
 	fmt.Printf("Backup request %q submitted successfully.\n", backup.Name)
+	if o.Wait {
+		fmt.Println("Waiting for backup to complete. You may safely press ctrl-c to stop waiting - your backup will continue in the background.")
+		ticker := time.NewTicker(time.Second)
+		defer ticker.Stop()
+
+		for {
+			select {
+			case <-ticker.C:
+				fmt.Print(".")
+			case backup, ok := <-updates:
+				if !ok {
+					fmt.Println("\nError waiting: unable to watch backups.")
+					return nil
+				}
+
+				if backup.Status.Phase != api.BackupPhaseNew && backup.Status.Phase != api.BackupPhaseInProgress {
+					fmt.Printf("\nBackup completed with status: %s\n", backup.Status.Phase)
```

Comment:
would be nice to also print something about using `ark backup describe` and `ark backup logs` for more information.